src/light.cpp

```diff
@@ -6,14 +6,14 @@
 Light::Light(std::string label, Type type, Transform transform, float power,
              float radius, glm::vec3 lightColor, bool castShadow)
     : SceneObject(SceneObject::LIGHT, label, transform),
-      m_Mesh(Importer::LoadFile("../assets/models/sphere.obj")), //
-      m_Type(type), m_Color(lightColor),                         //
-      m_Radius(radius), m_Power(power),                          //
-      m_InnerCone(20.0f), m_OuterCone(30.0f),                    //
-      m_NearPlane(1.0f), m_FarPlane(1000.0f),                    //
-      m_CastShadow(castShadow),                                  //
-      m_ShadowSize(1024),                                        //
-      m_ShadowTexture(nullptr) {
+      m_Mesh(Importer::LoadFile("../assets/models/sphere.obj")),       //
+      m_ShadowTexture(nullptr), m_ColorTexture(nullptr), m_Type(type), //
```

Comment:
```suggestion
      m_Type(type), m_ShadowTexture(nullptr), m_ColorTexture(nullptr), //
```
class members are initialized by the order of declaration in the header file